### PR TITLE
VIH-7190 return the room id to track VMR events from pexip

### PR DESCRIPTION
**Pull request by shaed-parkar**
### JIRA link (if applicable) ###

VIH-7190

### Change description ###

return the room id to track VMR events from pexip

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoApi/VideoApi/Controllers/ParticipantsController.cs [hunk 1]
```diff
@@ -312,6 +313,7 @@ public async Task<IActionResult> GetDistinctJudgeNamesAsync()
         [OpenApiOperation("GetParticipantsByConferenceId")]
         [ProducesResponseType(typeof(List<ParticipantSummaryResponse>), (int)HttpStatusCode.OK)]
         [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        [AllowAnonymous]
```
Comment (levibutler) [inline VideoApi/VideoApi/Controllers/ParticipantsController.cs:316]: nooope :P

File: VideoApi/VideoApi/VideoApi.csproj [hunk 2]
```diff
@@ -68,9 +68,9 @@
     </Content>
   </ItemGroup>
 
-  <Target Name="NSwag" AfterTargets="Build" Condition="'$(TF_BUILD)'!='True' AND '$(Configuration)'=='Debug'">
-    <Copy SourceFiles="@(ReferencePath)" DestinationFolder="$(OutDir)References" />
-    <Exec Command="$(NSwagExe_Core31) run /runtime:NetCore31 /variables:ProjectPath=&quot;$(MSBuildProjectFullPath)&quot;" />
-    <RemoveDir Directories="$(OutDir)References" />
-  </Target>
+<!--  <Target Name="NSwag" AfterTargets="Build" Condition="'$(TF_BUILD)'!='True' AND '$(Configuration)'=='Debug'">-->
```
Comment (levibutler) [inline VideoApi/VideoApi/VideoApi.csproj:71]: do you want to add this back in?

Comment (shaed-parkar) [reply VideoApi/VideoApi/VideoApi.csproj:71]: will do

File: VideoApi/VideoApi/Controllers/ParticipantsController.cs (hunk 1 above)
Comment (shaed-parkar) [reply VideoApi/VideoApi/Controllers/ParticipantsController.cs]: oh dear

File: VideoApi/VideoApi/ConfigureServicesExtensions.cs [hunk 3]
```diff
@@ -40,23 +41,18 @@ public static IServiceCollection AddSwagger(this IServiceCollection services)
...
+                document.AddSecurity("JWT", Enumerable.Empty<string>(),
+                    new OpenApiSecurityScheme
+                    {
+                        Type = OpenApiSecuritySchemeType.ApiKey,
+                        Name = "Authorization",
+                        In = OpenApiSecurityApiKeyLocation.Header,
+                        Description = "Type into the textbox: Bearer {your JWT token}.",
+                        Scheme = "bearer"
+                    });
+                document.Title = "Bookings API";
```
Comment (levibutler) [inline VideoApi/VideoApi/ConfigureServicesExtensions.cs:53]: bro lol

Comment (shaed-parkar) [reply VideoApi/VideoApi/ConfigureServicesExtensions.cs:53]: lol

**Review by levibutler — APPROVED** [review]

**Pull request merged**